dali/kernels/slice/slice_flip_normalize_permute_gpu.cuh

```diff
@@ -26,144 +26,27 @@
 #include "dali/kernels/common/copy.h"
 #include "dali/kernels/kernel.h"
 #include "dali/kernels/slice/slice_flip_normalize_permute_common.h"
-
+#include "dali/kernels/slice/slice_flip_normalize_permute_pad_kernel.cuh"
```

Comment:
I am thinking, since we have two concepts of kernel here (cuda kernel vs dali kernel) it'd be good to name this `*_cuda_impl.cuh` or something more clarifying than `*_kernel.cuh`